ksqldb-engine/src/main/java/io/confluent/ksql/execution/scalablepush/PushRouting.java

```diff
@@ -291,8 +291,8 @@ public void preparePushQuery(
       final String thisHostName
   ) {
     if (node.isLocal()) {
-      LOG.info("Query {} id {} executed locally at host {} at timestamp {}.",
-          statement.getStatementText(), pushPhysicalPlanManager.getQueryId(), node.location(),
+      LOG.info("Query with id {} executed locally at host {} at timestamp {}.",
```

Comment:
this was logging queries at INFO level which is a security violation - it also was logging ridiculously long strings of `aaaaaaaaaa` in the tests and clobbering my terminal so I was happy to get rid of it

Reply:
😆